gnd/src/main/java/com/google/android/gnd/persistence/local/LocalDataStore.java

```diff
@@ -44,6 +51,37 @@
  * deltas to allow changes to not rely on prior UI state (i.e., emissions are idempotent).
  */
 public interface LocalDataStore {
+
+  /** Add an option for a multiple choice field. */
+  Completable insertOrUpdateOption(String fieldId, Option option);
```

Comment:
Why does the LocalDataStore class need to expose these? Don't we only need the granular insert or update at the DAO level?

Reply:
Nice catch! Fixed